### PR TITLE
cog example (fixed code and RESTORED FILES)

### PR DESCRIPTION
**Pull request by yonipy**
## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by ysIgnacio** [comment]
3 prs

**Comment by yonipy** [comment]
im sorry first timer 🤷🏻‍♂️🤷🏻‍♂️😂😭

File: examples/cogs.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+def setup(client) -> commands.Cog:
+    test = commands.Cog("Test", "Some commands for testing.")  # You may edit the description however you want.
+
+    @test.command()
+    async def ping(ctx):
+        """Sends Pong!"""
+        await ctx.reply("Pong")
+
+    return test  # You returning the cogs.
```
Comment (EnokiUN) [inline examples/cogs.py:13]: English moment I suppose.

File: examples/cogs.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import voltage
```
Comment (EnokiUN) [inline examples/cogs.py:1]: Explain more stuff I suppose

**Review by EnokiUN — CHANGES_REQUESTED** [review]

File: examples/cogs.py (hunk 2 above)
Comment (yonipy) [reply examples/cogs.py:1]: ok…

**Comment by EnokiUN** [comment]
Can you make explanation not suck I suppose?

**Comment by yonipy** [comment]
😭😭😭 do i have to make a whole paragraph of this ???

**Comment by EnokiUN** [comment]
It has to be better than the cog example that I've made I suppose.

**Comment by EnokiUN** [comment]
@yonipy either fix this pr or just let me push my version of it, the choice is yours I suppose.

**Comment by EnokiUN** [comment]
:trollface:

**Comment by yonipy** [comment]
can i get tmr to do this im going to sleep now

**Comment by yonipy** [comment]
enoki i hate u sm

**Comment by EnokiUN** [comment]
Fine you can sleep I suppose

**Comment by EnokiUN** [comment]
Wait, can you get sued for depriving a person from sleep I suppose?

File: examples/cogs.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import voltage  # Import voltage.
+from voltage.ext import commands  # Importing the cog framework so we can use the cogs.
```
Comment (HigherOrderLogic) [inline examples/cogs.py:2]: Cog framework? :trollface:

Comment (ysIgnacio) [reply examples/cogs.py:2]: `from voltage.ext.commands import Cog` :bowtie:

Comment (EnokiUN) [reply examples/cogs.py:2]: # ***C O G  F R A M E W O R K***

I suppose.

Comment (ysIgnacio) [reply examples/cogs.py:2]: > # _**C O G F R A M E W O R K**_
> I suppose.

stop saying `I suppose` please.

Comment (yonipy) [reply examples/cogs.py:2]: Brun ok. ILL CHANGE IT TO COMMAND FRAMEWORK FFS u guys

Comment (EnokiUN) [reply examples/cogs.py:2]: > > # _**C O G F R A M E W O R K**_
> > I suppose.
> 
> stop saying `I suppose` please.

No I suppose :trollface:

**Comment by EnokiUN** [comment]
@b1ue64 @HigherOrderLogic @ysIgnacio how's this I suppose?

**Comment by yonipy** [comment]
u made a whole explaination ….

**Comment by ysIgnacio** [comment]
seems good ig

**Pull request merged**